examples/cfg/cfg.json

```diff
@@ -1,17 +1,17 @@
 {
-  "control": {},
-  "scheduler": {},
-  "plugins": {
-    "collector": {
-      "cinder": {
-        "all": {
-          "endpoint": "https://public.fuel.local:5000",
-          "user": "admin",
-          "password": "admin"
+  "control": {
+    "plugins": {
+      "collector": {
+        "cinder": {
+          "all": {
+            "endpoint": "http://localhost:5000/v2",
```

Comment:
I meant keystone endpoint. I know `public.fuel.local` doesn't exist, but is says more than `localhost`. It would be even better to change it to `keystone.endpoint.local`.